test/support/utils/TaxYearUtils.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package support.utils
+
+import java.time.LocalDate
+
+object TaxYearUtils {
+
+  private val dateNow: LocalDate = LocalDate.now()
+  private val taxYearCutoffDate: LocalDate = LocalDate.parse(s"${dateNow.getYear}-04-05")
```

Comment:
A minor one, maybe we can use `LocalDate.of(year, month, day)` instead? Parsing feels a bit unnecessary here? Or is there a particular reason?

Reply:
I am in favor of using LocalDate.of(...), however if you use it you will have to use Ints which will give a magic number warning. Parsing is a method that comes from Java library, so I don't see a big problem for using it.
Do you have a better suggestion?

Reply:
Strangely enough library usages is not always encouraged. Even between our own modules we often copy/paste code. I asked the same question more than a year ago.